libsplinter/src/admin/rest_api/resources/circuits.rs

```diff
@@ -12,23 +12,62 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::circuit::{AuthorizationType, DurabilityType, PersistenceType, Roster, RouteType};
+use std::collections::BTreeMap;
+
+use crate::circuit::{Circuit, Roster, ServiceDefinition};
 use crate::rest_api::paging::Paging;
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub(crate) struct ListCircuitsResponse {
```

Comment:
Likewise, it doesn't need to own its data

Reply:
Updated to borrow when possible